wrench/src/yaml_helper.rs

```diff
@@ -145,6 +145,15 @@ impl YamlHelper for Yaml {
     }
 
     fn as_matrix4d(&self) -> Option<LayoutTransform> {
+        if let Some(mut nums) = self.as_vec_f32() {
```

Comment:
I'd like to keep this with the if because I want to add support for short hand like "rotate(45deg)" or "translate(1, 2)"